doc/design/design_model_zoo.md

```diff
@@ -0,0 +1,187 @@
+# Model Zoo
+
+SQLFlow model zoo is a place to store model definitions, pre-trained model parameters (or weights) and model documentations. You can directly train, predict, analyze using one of the models using SQLFlow, or you can do model fine-tune, transfer learning to use the model to fit your dataset.
+
+SQLFlow should support below features to support common cases in machine learning:
+
+1. Host model definition and pre-trained parameters in `sqlflow.org`. e.g. `sqlflow.org/modelzoo/iris_dnn_128x32` points to a directory containing a model definition of `DNNClassifier` with 128, 32 hidden layers and pre-trained parameters using the iris dataset.
```

Comment:
I think the model zoo contains not only the "model", but also the process that converts data into model inputs. Only if we have both parts in the model zoo, we could make sure that the prediction and the training are using the same conversion from data to model inputs.

Reply:
Input conversion information is included at: https://github.com/sql-machine-learning/sqlflow/pull/949/files#diff-4979133b9186419f2057f96cafe9b0c1R63